Doc/library/sqlite3.rst

```diff
@@ -486,18 +509,31 @@ Connection Objects
 
    .. method:: create_function(name, narg, func, *, deterministic=False)
 
-      Creates a user-defined function that you can later use from within SQL
-      statements under the function name *name*. *narg* is the number of
-      parameters the function accepts (if *narg* is -1, the function may
-      take any number of arguments), and *func* is a Python callable that is
-      called as the SQL function. If *deterministic* is true, the created function
-      is marked as `deterministic <https://sqlite.org/deterministic.html>`_, which
-      allows SQLite to perform additional optimizations. This flag is supported by
-      SQLite 3.8.3 or higher, :exc:`NotSupportedError` will be raised if used
-      with older versions.
+      Create a user-defined SQL function.
+
+      :param name:
+          The name of the SQL function.
+      :type name: str
+
+      :param narg:
+          The number of parameters the SQL function can accept.
+          If ``-1``, it may take any number of arguments.
+      :type narg: int
 
-      The function can return any of
-      :ref:`the types natively supported by SQLite <sqlite3-types>`.
+      :param func:
+          A callable that is called when the SQL function is invoked.
+          The callable must return :ref:`a type natively supported by SQLite
+          <sqlite3-types>`.
+          Set to :const:`None` to remove the SQL function.
+      :type func: :term:`callback` | :const:`None`
```

Comment:
I'm not sure if using `:term:callback` is the best reference here. Ideally, there should be a `:term:callable`.

Reply:
I suggest we add a `:term:` for callable first; then we can pull in `main` and continue with this PR.

Reply:
OTOH, we can just land this and update the docs once there is a `:term:` for callable.